mfthomps/CyberCIEGE-port-to-Unity3d
# Merge Request for Issue/16 - buy mode visuals

Closes #16 

To test:
- Run any scenario that has hardware available to buy (like introductory scenario)
- After clicking on buy button in Buy View, a mesh of the hardware being bought should follow the cursor
- A yellow square should show up in the workspace the hardware will be put in
- If you repeat this, the workspace square will turn red if the workspace can't have more hardware of the given type
